### PR TITLE
Fix lint test runner, don't need repeats per python

### PR DESCRIPTION
**Pull request by carver**
It's too easy to incorrectly set up multiple lint tests in tox.ini, so it's
better to just have one. Not much gained from re-running per python
anyway.

#### Cute Animal Picture

put a cute animal picture link inside the parentheses

### CODE REVIEW

**Comment by carver** [comment]
Sorry, misclick. Meant to ping for review @pipermerriam

File: eth_utils/curried/__init__.pyi [hunk 1]
```diff
@@ -1,86 +1,5 @@
...
-    to_int,
-    to_list,
-    to_normalized_address,
-    to_ordered_dict,
-    to_set,
-    to_text,
-    to_tuple,
-    to_wei,
-)
-
```
Comment (pipermerriam) [inline eth_utils/curried/__init__.pyi:83]: Why was this removed?  Won't this break all of the `from eth_utils.curried` imports?

Comment (carver) [reply eth_utils/curried/__init__.pyi:83]: (notice this is the `pyi` file, not the `py` file)

mypy was failing with warnings, on that import. I don't really know the pyi format, but no tests broke when I removed it. :man_shrugging: 

Here's the test with all the warnings:
https://circleci.com/gh/ethereum/eth-utils/2623?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

From my quick read of https://www.python.org/dev/peps/pep-0484/#stub-files I think there's no reason/need for those imports to be there.
